umap/static/umap/js/modules/sync/engine.js

```diff
@@ -1,7 +1,7 @@
+import * as Utils from '../utils.js'
+import { HybridLogicalClock } from './hlc.js'
```

Comment:
Comment unrelated to this PR, but this "biome noise" is a bit of a pity, while the main scope of biome in my understanding is to remove all formatting noise from PR. How is it that this passed the CI ? @davidbgk any idea ?